modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -414,9 +338,7 @@ If you want to delete an SSL certificate, use the delete.sh script, running it a
 sudo /etc/letsencrypt/delete.sh
 ----
 
-It will start off, as below, by displaying a list of the currently
-available SSL certificate domain names, and then prompt you to supply
-the certificate that you want to delete.
+It will start off, as below, by displaying a list of the currently available SSL certificate domain names, and then prompt you to supply the certificate that you want to delete.
```

Comment:
```suggestion
It will start off by displaying a list of the currently available SSL certificate domain names, as in the example below, and then prompt you to supply the certificate you want to delete.
```